lock/lock_test.go

```diff
@@ -0,0 +1,55 @@
+package lock
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/go-commons/logging"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBlockingAcquireLock(t *testing.T) {
+	t.Parallel()
+
+	lockString := "guardduty-blocking-acquire-lock-test"
+	maxRetries := 2
+	lockTable := "test-dynamo-lock-eu-jam"
```

Comment:
Who creates the table?

Reply:
Some thoughts from discussing:
- so far it's been done manually 
- we want to create a table in a consistent way, only in one central location, one region

the plan is: 
- add a new interface to allow creation of a dynamoDB table with these params.